src/App.js

```diff
@@ -0,0 +1,39 @@
+import './App.css';
+import React, { useState, useEffect } from 'react';
+
+function App() {
+  const [data, setData] = useState([]);
+  const [searchTerm, setSearchTerm] = useState([]);
+  const getDataFromAPI = () => {
+    fetch('https://swapi.dev/api/people/?search=sky').then((response) => {
+      return response.json()
+    }).then((response) => {
+      console.log(response.results)
+      for (var i = 0; i < response.results.length; i++) {
+        data.push(response.results[i].name)
+      }
+      setData(data);
+    })
+  }
```

Comment:
this could've been another component, but ideally should be part of the backend server